src/ui/PrintableUri/PrintableUri.ts

```diff
@@ -12,7 +12,9 @@ export interface IPrintableUriOptions {
 }
```

Comment:
This component has several unexposed public methods (renderParentsXml, renderUri, buildSeparator, buildEllipsis and buildHTMLToken). Would you like me to expose some of those in the public doc?